---
layout: default
title: Research
permalink: /research
---

# Multi-omics integration<br>
My previous research has focused on genome-wide annotation of genomic elements and linking genetic effects to gene expression and organismal phenotypes. Building on this work, my future research will aim to elucidate genetic effects on various molecular and organismal traits at higher resolution (e.g., single-cell/nuclei level) in diverse contexts. This will provide deeper insights into the genetic and molecular basis of agriculturally significant complex phenotypes in farm animals.<br> 
## Discover context-specific epigenomic regulatory elements in farm animal genomes. 
This project will collect tissue samples from Holstein cows and perform ATAC-Seq to annotate active regulatory elements at single-cell resolution across key developmental stages. This research will generate a map of developmental stage-specific regulatory elements across diverse tissues and assess their impacts on milk production traits in dairy cows.<br>
## Develop computational tools for missing value imputation in farm animals. 
Given the sparsity of epigenomic data, especially at the single-cell level, this project will develop deep learning methods to enable cross-context epigenome imputation by leveraging correlations from existing experimental data. The developed approach will complement wet-lab experiments, computationally generating complete animal epigenomes across diverse contexts.<br>
## Identify genetic effects on molecular traits. 
This work will emphasize the identification of genetic effects interacting with environmental factors, such as developmental stage, ancestry, and sex-specific effects. Looking ahead, I plan to integrate additional omics technologies to profile other layers of gene regulation, including epigenetic modifications (e.g., DNA methylation, histone modifications), protein abundance, and metabolite levels.<br>  
<br>
# Comparative genomics
Phenotypic traits such as stature and body weight are common across farm animal species, as well as in humans. Comparative genomics offers a powerful approach to studying the conservation of genetic and molecular mechanisms, advancing our understanding of human biology and diseases like developmental disorders, metabolic syndrome, and cardiovascular disease. With my background in working on multiple species and expertise in population and comparative genomics, a key focus of my future research will be conducting comparative genomics to deepen our understanding of the evolutionary processes shaping molecular and phenotypic diversity within and across species.<br>
## Track the evolution of complex phenotypes during animal domestication and selective breeding. 
The animal Quantitative Trait Loci (QTL) database and the Farm Animal GTEx (FarmGTEx) project offer valuable resources for studying the evolution of complex phenotypes. In this project, I will use ancient DNA (aDNA) to reconstruct endophenotypes (e.g., gene expression, splicing, chromatin interaction) from archaeological and historical chicken genomes, examining their evolutionary trajectories. This multi-disciplinary work integrates quantitative genetics, population genomics, and paleogenetics to explore the within-species evolution of complex phenotypes in chickens.<br>
## Investigate the epigenetic basis of phenotypic diversity across species. 
Leveraging data from international consortia like ENCODE, Roadmap Epigenomics, FAANG, GTEx projects, I will study how epigenomic variation, such as enhancers, relates to phenotypic variation across species. In this project, I will train deep learning models to predict enhancers across species and explore enhancer-phenotype associations. This innovative work combines cutting-edge deep learning methodologies to study the cross-species evolution of complex phenotypes.<br>








